### DISCUSSION TITLE
chave ssh publica não é gerada

### DISCUSSION THREAD

**Discussion by Davi-pontes** [community/community#50737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

estou tentando gerar minha chave ssh para acessar o github pelo computador, porém só está sendo gerada minha senha privada a senha publica não está sendo gerada.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @Davi-pontes ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
